projects/common/src/lib/state/iot-ensemble-state.context.ts

```diff
@@ -117,7 +117,7 @@ export class IoTEnsembleStateContext extends StateContext<IoTEnsembleState> {
   public UpdateTelemetrySync(refreshRate: number, pageSize: number) {
     this.Execute({
       Arguments: {
-        RefreshRate: refreshRate,
+        RefreshRate: 30,

```

Comment:
Move this 30 up one level to where UpdateTelemetrySync is being called.  Should be in dynamic.component.ts in app-host.  this will allow us to keep the api working if we ever do want to change refresh rate, and not as hard coded.